contents/docs/surveys/installation.mdx

```diff
@@ -7,8 +7,13 @@ availability:
     selfServe: full
     enterprise: full
 ---
-import WebInstall from '../integrate/_snippets/install-web.mdx'
 
-To get started with surveys, first install PostHog in your app:
+Surveys are a great tool to collect qualitative feedback from your users.
+
+To get started with surveys, add PostHog to your web app by installing our **JavaScript snippet** or **`posthog-js` library**:
```

Comment:
Should this be JavaScript Web SDK for consistency? Mention React SDK? I'm not sure.